File: app/src/main/java/org/stepic/droid/storage/structure/DbStructureBlock.java
package org.stepic.droid.storage.structure;

public final class DbStructureBlock {
    public static final String BLOCKS = "blocks";

    public static final class Column {
        public static final String STEP_ID = "step_id";
        public static final String NAME = "name";
        public static final String TEXT = "block_text";

        public static final String EXTERNAL_THUMBNAIL = "optional_thumbnail";
        public static final String EXTERNAL_VIDEO_ID = "optional_video_id";
    }
}

File: app/src/main/java/org/stepic/droid/storage/structure/DbStructureBlock.kt
package org.stepic.droid.storage.structure

object DbStructureBlock {

    const val BLOCKS = "blocks"

    object Column {
        const val STEP_ID = "step_id"
        const val NAME = "name"
        const val TEXT = "block_text"

        const val EXTERNAL_THUMBNAIL = "optional_thumbnail"
        const val EXTERNAL_VIDEO_ID = "optional_video_id"
        const val EXTERNAL_VIDEO_DURATION = "optional_video_duration"


        //we won't use some properties for getting the code option from database
        //and we can store the rest of the object as a string
        const val CODE_OPTIONS = "code_options"
    }

}